### ISSUE TITLE
test issue

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/cdisc-rules-engine#1632]
### Standard

CDISC SEND, CDISC SDTM/SDTMIG

### Reference Rule ID(s)

cg000099

### Conformance Rule ID(s) (if published in CORE)

CORE-000123

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### Rule YAML

```YAML
Authorities: 
  - Organization: 
    Standards:
      - Name: 
        References:
          - Citations:
              - Cited Guidance: 
                Document: 
                Item: 
                Section: 
            Origin: 
            Rule Identifier: 
              Id: 
              Version: 
            Version: 
        Version:
Check:
  all:
    - name: 
      operator: 
      value: 
Core:
  Id: 
  Status: 
  Version: '1'
Description: 
Executability: Fully Executable
Match Datasets:
  - Keys:
      - 
    Name: 
Operations:
  - id: 
    operator: 
Outcome:
  Message: 
  Output Variables:
    -
Rule Type: 
Scope:
Sensitivity:
```

### Sample Test Data File

test data file

### Output report and/or log file

output report

### Expected output

expected output

### Any Additional Information

some more notes
